modules/src/ics03_connection/handler/verify.rs

```diff
@@ -197,10 +197,7 @@ pub fn check_client_consensus_height(
         return Err(Kind::InvalidConsensusHeight(claimed_height, ctx.host_current_height()).into());
     }
 
-    let oldest_available_height =
-        ctx.host_current_height().revision_height - ctx.host_chain_history_size() as u64;
```

Comment:
Can we remove method `host_chain_history_size`, which is no longer useful now that you introduced `host_oldest_height`?

Reply:
Done in c851c336c039286ef803cdae5e693d058cf0c342.